Hardware/12-Wifi_Programming_Tutorial.md

```diff
@@ -3,9 +3,29 @@
 **The Wifi shield and these docs are still in beta, if you see a typo [please open an issue](https://github.com/OpenBCI/Docs/issues/new).**
```

Comment:
Should't the title be WiFi uploading code to the shield?

Reply:
i remember us talking about this but i don't remember the outcome